docs/source/user-guide/nlp/supported_tasks.rst

```diff
@@ -22,28 +22,31 @@ Deepchecks currently supports two NLP task types:
   sample. Examples for such tasks are: Sentiment Analysis, Topic Extraction, harmful content detection and more.
 * **Token Classification**: Token Classification is any NLP task in which each word (or to be more accurate, token) in
```

Comment:
Can you add links here and in the previous to the relevant sections at the bottom?